### PR TITLE
[Security] Fix annotation

### PR DESCRIPTION
**Pull request by enumag**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Authentication/Token/UsernamePasswordToken.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class UsernamePasswordToken extends AbstractToken
      * Constructor.
      *
      * @param string|object            $user        The username (like a nickname, email address, etc.), or a UserInterface instance or an object implementing a __toString method
-     * @param string                   $credentials This usually is the password of the user
+     * @param mixed                    $credentials This usually is the password of the user
```
Comment (dmaicher) [inline src/Symfony/Component/Security/Core/Authentication/Token/UsernamePasswordToken.php]: why would it not be a `string`?

Comment (enumag) [inline src/Symfony/Component/Security/Core/Authentication/Token/UsernamePasswordToken.php]: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Core/Authentication/Token/TokenInterface.php#L40-L45

In my app it is null in some cases like impersonation.

Comment (fabpot) [inline src/Symfony/Component/Security/Core/Authentication/Token/UsernamePasswordToken.php]: Then string|null?

**Review by fabpot — CHANGES_REQUESTED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @enumag.

**Pull request merged**